gradoop-examples/src/main/java/org/gradoop/utils/centrality/DegreeCentralityExample.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright © 2014 - 2018 Leipzig University (Database Research Group)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.gradoop.utils.centrality;
+
+import org.apache.flink.api.common.ProgramDescription;
+import org.apache.flink.api.java.DataSet;
+import org.gradoop.examples.AbstractRunner;
+import org.gradoop.flink.model.api.epgm.LogicalGraph;
+import org.gradoop.flink.model.impl.operators.sampling.statistics.DegreeCentrality;
+
+/**
+ * Example class calculating degree centrality of a graph
+ */
+public class DegreeCentralityExample extends AbstractRunner implements ProgramDescription {
```

Comment:
Since you already implemented the `DegreeCentralityRunner` this example class is not necessary. You can remove it and the additional package `centrality`.